addons/hr_holidays_attendance/static/src/components/accrual_levels/accrual_levels.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-name="hr_holidays_attendance.AccrualLevels" t-inherit="hr_holidays.AccrualLevels" t-inherit-mode="extension">
+        <xpath expr="//h4[hasclass('header')]" position="inside">
```

Comment:
use a name attribute on the base element to target it more explicitely, this might be error-prone depending on how the view evolves